src/views/AnnualReport.vue

```diff
@@ -990,7 +990,7 @@ export default class AnnualReport extends Vue {
 
     if (this.isCoop) {
       annualReport = {
-        annualReport: {
+        [FilingTypes.ANNUAL_REPORT]: {
```

Comment:
"Better" code since it's more generic, even though the old code worked. But I think it does emphasize the point that this data object is named as per the filing type.